db/migrate/20160119125950_add_created_on_for_container_entities.rb

```diff
@@ -1,15 +1,44 @@
 class AddCreatedOnForContainerEntities < ActiveRecord::Migration
-  CONTAINER_TABLES = [:container_nodes, :container_projects, :container_services, :container_routes, :container_groups,
-                      :container_replicators, :container_quotas, :container_builds, :container_build_pods,
-                      :container_limits, :container_volumes]
+  class ContainerNode < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class ContainerProject < ActiveRecord::Base; end
```

Comment:
No.

Part of the advantage of ApplicationRecord is that we can leave AR::Base unmolested, which is exactly what migrations want: a plain ordinary AR object with no app-code-of-the-day surprises.